backend/sirius-web-graphql/src/main/java/org/eclipse/sirius/web/graphql/datafetchers/mutation/MutationUpdateWidgetFocusDataFetcher.java

```diff
@@ -44,7 +45,7 @@
 // @formatter:off
 @GraphQLMutationTypes(
     input = UpdateWidgetFocusInput.class,
-    payloads = {
+    payloads = {UpdateWidgetFocusSuccessPayload.class
```

Comment:
Nitpick: should be on a new line, see the other examples like
```java
    payloads = {
        UploadDocumentSuccessPayload.class,
    }
```

Reply:
Done